internal/core/src/common/ChunkTarget.cpp

```diff
@@ -0,0 +1,42 @@
+#include <common/ChunkTarget.h>
+#include <cstring>
+#include "common/EasyAssert.h"
+#include <sys/mman.h>
+
+namespace milvus {
+void
+MemChunkTarget::write(const void* data, size_t size) {
+    AssertInfo(size + size_ <= cap_, "can not exceed target capacity");
+    std::memcpy(data_ + size_, data, size);
+    size_ += size;
+}
+
+std::pair<char*, size_t>
+MemChunkTarget::get() {
+    return {data_, size_};
+}
+
+size_t
+MemChunkTarget::tell() {
```

Comment:
Any specific reason to name this function `tell` rather than `size`?

Reply:
> Any specific reason to name this function `tell` rather than `size`?

tell is a conventional name to tell caller how much space have been used